CoffeeChat/Layouts/LeftAlignedFlowLayour.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  LeftAlignedFlowLayour.swift
+//  CoffeeChat
+//
+//  Created by Lucy Xu on 10/18/20.
+//  Copyright © 2020 cuappdev. All rights reserved.
+//
+
+import UIKit
+
+/// LeftAlignedFlowLayout aligns all collection items to the leftmost position
+class LeftAlignedFlowLayout: UICollectionViewFlowLayout {
+
+    override func layoutAttributesForElements(in rect: CGRect) -> [UICollectionViewLayoutAttributes]? {
+        guard let originalAttributes = super.layoutAttributesForElements(in: rect) else {
+            return nil
+        }
+        var leftMargin: CGFloat = 0.0
+        var lastY: Int = 0
+        return originalAttributes.map {
+            let changedAttribute = $0
+            // Check if start of a new row.
+            // Center Y should be equal for all items on the same row
+            if Int(changedAttribute.center.y.rounded()) != lastY {
```

Comment:
Do you need to round? If flooring is enough I'd just do the int cast for the sake of simplicity.